### DISCUSSION TITLE
Accept Offer on behalf of seller at the end of auction

### DISCUSSION THREAD

**Discussion by Dashon** [ProjectOpenSea/seaport-js#344] [General]
I'm implementing auctions where the highest offer would be accepted automatically at the end of the auction. I planned to do this off-chain via a cronJob. Like Opensea, I would like to fulfill the offer on behalf of the seller and pay the gas fee. How can I do this? Do I need to utilize zones and or conduits?

**Comment by montasaurus** [comment]
You call `matchAdvancedOrders` with the corresponding listing and top bid. Here's an example transaction: https://etherscan.io/tx/0xeed31a5401ffb80f396bdca493ed49ec692dc77871a395356b4483ef1ee21bcd
